src/screens/users/jobs/show/index.jsx

```diff
@@ -27,21 +28,23 @@ const Wrapper = ({ title, children }) => {
 export const JobScreen = ({ job }) => {
   const { isAuthenticated, person } = useAuth()
   const { tokenSymbol } = useContext(Web3Context)
+  const { publicRuntimeConfig } = getConfig()
+  const testDomain = publicRuntimeConfig.domain
```

Comment:
What do you think about `domain` for variable name? `testDomain` looks a bit weird 😉